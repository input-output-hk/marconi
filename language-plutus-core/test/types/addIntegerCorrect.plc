(program 0.1.0
  [
    (lam x [(con integer) (con 2)]
      [{(con addInteger) (fun (fun [(con integer) (con 2)] [(con integer) (con 2)]) [(con integer) (con 2)])} x (con 2 ! 2)])
    (con 2 ! 2)
  ]
)
