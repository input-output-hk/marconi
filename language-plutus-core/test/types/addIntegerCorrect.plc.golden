TyApp () (TyBuiltin () TyInteger) (TyInt () 2 :| [])